Test/XLTest.cs

```diff
@@ -1168,7 +1168,8 @@ public static void TestWriteToPercolator()
                 XlSearchParameters = new XlSearchParameters
                 {
                     WriteOutputForPercolator = true
-                }
+                },
+                CommonParameters = new CommonParameters(trimMsMsPeaks: false)
```

Comment:
I added peak filtering for .mgf files to the new mzLib version, which was missing. I edited this unit test to not filter MS2 peaks to keep the results of the test the same as the old version.